src/Migration/Migration1607529255AdyenNotification.php

```diff
@@ -15,8 +15,8 @@ public function getCreationTimestamp(): int
     public function update(Connection $connection): void
     {
         $connection->executeUpdate("
-            alter table `adyen_notification`
-            add column scheduled_processing_time datetime null after processing;
+            ALTER TABLE `adyen_notification`
+            ADD COLUMN `scheduled_processing_time` DATETIME(3) NULL AFTER `processing` COMMENT 'Scheduled for';
```

Comment:
Why `DATETIME(3)`? And would this be applied in instances where this migration was already applied? Or should we move it to a new migration?

Reply:
I made this optional change in a separate commit because it is only about being consistent with the other migration `\Adyen\Shopware\Migration\Migration1589892467AdyenNotification` which creates the table.
I can only still change this because this branch `PW-3049` is not yet merged into `develop` and therefore this is still a new migration. If you have already run this migration in your VM then no it will not change anything.

Reply:
Ah that's right, this hasn't been released yet 😂